ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.15__jenkins-ocp4.15-lp-interop.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
```

Comment:
Update version from 4.13

Reply:
see response here: https://github.com/openshift/release/pull/44988#discussion_r1376808230

if someone who knows this scenario well and can offer if this image can be updated please let us know.